src/js/main.js

```diff
@@ -32,6 +33,163 @@ Site = {
+          month: month,
+          days: []
+        };
+      }
+
+      // Create this Day object if missing in Month
+      if (_this.scheduleArray[year].months[monthNum].days[dayNum] === undefined) {
+        _this.scheduleArray[year].months[monthNum].days[dayNum] = {
+          day: day,
+          events: []
+        };
+      }
+
+      // Push Event object to events array
+      _this.scheduleArray[year].months[monthNum].days[dayNum].events.push({
+        hour: hour,
+        title: el.title,
+        content: el.content
+      });
+    });
+
+    //
+    _this.addEventsToDom();
+  },
+
+  addEventsToDom: function() {
+    var _this = this;
+
+    // Make Years
+    _this.scheduleArray.forEach(function(year, yearIndex) {
+
+      // create content element
+      var yearContent = document.createElement('h3');
+      // fill with year
+      yearContent.innerText = year.year;
+
+      // make Year row
+      _this.makeEventsRow(
+        // parent element
+        _this.programacionContainer,
+        // row id
+        yearIndex,
+        // row content
+        yearContent
+      );
+
+      // Make Months
+      year.months.forEach(function(month, monthIndex) {
+
+        // create content element
+        var monthContent = document.createElement('h3');
+        // fill with month
+        monthContent.innerText = month.month;
+
+        // make Month row
+        _this.makeEventsRow(
+          document.getElementById(yearIndex),
+          yearIndex + '-' + monthIndex,
+          monthContent
+        );
+
+        // Make Days
+        month.days.forEach(function(day, dayIndex) {
+
+          // create content element
+          var dayContent = document.createElement('h3');
+          // fill with day
+          dayContent.innerText = day.day;
+
+          // make Day row
+          _this.makeEventsRow(
+            document.getElementById(yearIndex + '-' + monthIndex),
+            yearIndex + '-' + monthIndex + '-' + dayIndex,
+            dayContent
+          );
+
+          // Make Events
+          day.events.forEach(function(eventObj, eventIndex) {
+
+            // create content element
+            var eventContent = document.createElement('div');
+            // set element element class
+            eventContent.className = 'programacion-event';
+            // fill with event info
+            eventContent.innerHTML = '<div class="programacion-event-hour">' + eventObj.hour + '</div><div class="programacion-event-info"><h2>' + eventObj.title + '</h2>' + eventObj.content;
+
+            // make Event row
+            _this.makeEventsRow(
+              document.getElementById(yearIndex + '-' + monthIndex + '-' + dayIndex),
+              yearIndex + '-' + monthIndex + '-' + dayIndex + '-' + eventIndex,
+              eventContent
+            );
+
+          });
+
+        });
+
+      });
+
+    });
+  },
+
+  makeEventsRow: function(container, id, rowContent) {
+    var _this = this;
```

Comment:
hey were going to leave this asis for now. we checked out performance and it isn't too heavy. i'm adding an issue to improve it as an enhancement.